File: 1 - Data Types/Solution.fs
open System

[<EntryPoint>]
let main argv = 
    let i = 4
    let d = 4.0
    let s = "HackerRank "
    let i2 = Console.ReadLine() |> int
    let d2 = Console.ReadLine() |> float
    let s2 = Console.ReadLine() |> string
    printfn "%i" (i + i2)
    printfn "%f" (d + d2)
    printfn "%s" (s + s2)
    0

File: 1 - Data Types/Solution.ts
import readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const getline = () => {
  return new Promise(resolve => {
    rl.question("", resolve);
  });
};

const i = 4;
const d = 4.0;
const s = "HackerRank ";

(async () => {
  const i2 = await getline();
  const d2 = await getline();
  const s2 = await getline();
  console.log(i + parseInt(i2.toString()));
  console.log(d + parseFloat(d2.toString()));
  console.log(s + s2.toString());
  rl.close();
})();
